app/portainer/services/fileUpload.js

```diff
@@ -28,6 +28,22 @@ angular.module('portainer.app')
     });
   };
 
+  service.loadImages = function(file) {
+    var endpointID = EndpointProvider.endpointID();
+    Upload.setDefaults({ ngfMinSize: 10 });
```

Comment:
I think that you should rather use the `ngf-min-size attribute` on the input with the `ngf-select` tag rather than overriding the defaults (it will set this value for all the other consecutive calls to `Upload`).